cmd/root.go

```diff
@@ -46,13 +47,16 @@ func Execute() {
 	const threads = 2
 	wg.Add(threads)
 
+	ctx, cancel := context.WithCancel(context.Background())
+
 	go func() {
-		sendAnalytics()
+		sendAnalytics(ctx)
 		wg.Done()
 	}()
 
 	go func() {
 		err = rootCmd.Execute()
+		cancel()
 		wg.Done()
```

Comment:
```suggestion
		defer wg.Done()
		defer cancel()
		err = rootCmd.Execute()
```

Without this change, the call to `cancel()` is not guaranteed.

Reply:
> not guaranteed.

🤔 In which scenario will that fail?

Reply:
If the control flow of the call stack gets interrupted for example by an ongoing `panic`.

Reply:
My bad. I had a wrong understanding of the relationship between `panic` and `defer`.